src/styles/blocks/month.scss

```diff
@@ -0,0 +1,59 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  align-self: center;
+  width: getWidth($day-size, $month-paddings, $days-per-week, $days-gap);
+  gap: $days-gap;
+  margin: 0 auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.